Kickstarter-iOS/Views/Controllers/ProjectPageViewControllerTests.swift

```diff
@@ -234,6 +302,40 @@ internal final class ProjectPageViewControllerTests: TestCase {
     let liveProject = self.project
       |> Project.lens.stats.convertedPledgedAmount .~ 1_964
       |> Project.lens.rewardData.rewards .~ []
+      |> \.extendedProjectProperties .~ self.projectProperties
+
+    let projectPamphletData = Project.ProjectPamphletData(project: liveProject, backingId: nil)
+
+    let mockService = MockService(
+      fetchManagePledgeViewBackingResult: .success(.template),
+      fetchProjectPamphletResult: .success(projectPamphletData),
+      fetchProjectRewardsResult: .success(self.project.rewards)
+    )
+
+    combos(Language.allLanguages, [Device.phone4inch, Device.pad]).forEach { language, device in
+      withEnvironment(
+        apiService: mockService,
+        config: config, currentUser: nil, language: language
+      ) {
+        let vc = ProjectPageViewController.configuredWith(projectOrParam: .left(liveProject), refTag: nil)
+
+        let (parent, _) = traitControllers(device: device, orientation: .portrait, child: vc)
+        parent.view.frame.size.height = device == .pad ? 1_200 : parent.view.frame.size.height
+
+        scheduler.run()
+
+        FBSnapshotVerifyView(vc.view, identifier: "lang_\(language)_device_\(device)")
+      }
+    }
+  }
+
+  func testLoggedOut_NonBacker_LiveProject_EmptyProjectProperties() {
```

Comment:
I think when we fix these tests to render content, we'll not use a single "all is empty" `extendedProjectProperties`, but instead go with each tab being populated and empty (2 test cases per tab).
Imo we can rename this test `testLoggedOut_NonBacker_LiveProject_EmptyProjectProperties` and `testLoggedIn_Backer_LiveProject_EmptyProjectProperties` by appending `_renderedEnvironmentalCommitments_Success`